### ISSUE TITLE
[Suggestion] Ajout d'un bouton de récupération de vote en attentes pour chacun des serveurs.

### ISSUE THREAD

**Issue by ghost** [MineWeb/Plugin-Vote#62]
Bonjour,

Je voulais savoir si il était possible d'ajouter un bouton de récupération pour les votes en attentes pour chacun des serveurs, Car par exemple sur mon serveur actuellement quand un joueurs veut récupérer ces votes du Skyblock et Survie il doit switch entre ces deux derniers ce qui est très vite chiant pour le joueur en question.

Cordialement,
AFGAME

**Comment by nivcoo** [comment]
C'est déjà le cas dans la dernière version du plugin

**Comment by ghost** [comment]
Est-til fonctionnel pour tout les thèmes ?

**Comment by nivcoo** [comment]
Normalement oui ^^

**Comment by ghost** [comment]
D'accord merci 👍

**Comment by ghost** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/50768722/80258159-d1af0480-8682-11ea-95d1-280208157b0c.png)
![Screenshot_1](https://user-images.githubusercontent.com/50768722/80258160-d2479b00-8682-11ea-8068-29299a81140d.png)
![Screenshot_2](https://user-images.githubusercontent.com/50768722/80258161-d2479b00-8682-11ea-9ea1-14ab731c7ba9.png)
Voilà en gros je te montre chaque étape, j'ai voter deux fois le survie et deux fois le skyblock, et même en ayant le plugin à jour il m'affiche toujours qu'un bouton dans le profil.

**Comment by nivcoo** [comment]
C'est sur ton profile que tu peux tout recup d'un coup

**Comment by ghost** [comment]
Oui, mais ce que je demandais était est-ce que l'on peut avoir deux ou plusieurs boutons de récupération pour chacun des serveurs ?

**Comment by ghost** [comment]
![Screenshot_4](https://user-images.githubusercontent.com/50768722/80276089-5da64800-86e6-11ea-8f6d-b6704f09b711.png)
J'avais oublié un screen

**Comment by nivcoo** [comment]
Non, dans tous les cas ca recup toute les récompenses d'un serveur après ça passe à l'autre

**Comment by ghost** [comment]
Donc il n'y à pas un ou plusieurs bouton pour cela ? Donc est-ce qu'il serait possible pour une futur maj de l'ajouter, car cela est vite relou d'aller sur le survie puis après skyblock, car enfaîte cela dépend de comment on à voté imaginons j'ai alternée entre chaque je vais devoir faire survie,skyblock,survie....
C'est pour cela qu'ajouté des boutons en fonction du serveur serait cool

**Comment by nivcoo** [comment]
Non tu comprends pas, que tu vote en alterné ou non ça donne toute les récompenses d'un serveur en même temps
